charts/publisher/templates/web/ingress.yaml

```diff
@@ -23,11 +24,11 @@ spec:
     {{- end }}
   {{- end }}
   rules:
-    - host: {{ .Values.ingress.host | default (printf "publisher-%s.eks.%s.%s" .Release.Namespace .Values.govukEnvironment .Values.govukDomainExternal ) }}
+    - host: {{ .Values.web.ingress.host | default (printf "publisher-%s.eks.%s.%s" .Release.Namespace .Values.common.govukEnvironment .Values.common.govukDomainExternal ) }}
       http:
         paths:
-          - path: {{ .Values.ingress.path }}
-            pathType: Prefix
+          - path: {{ .Values.web.ingress.path }}
+            pathType: {{ .Values.web.ingress.pathType }}
```

Comment:
Would there be a use case for `Exact`, given the application? (I can't think of one.) Or is the idea to support potential future usage of `ImplementationSpecific` with a different ingress controller?

Reply:
I'm not sure if there'll ever be a case for a pathType other than `Prefix` as it depends on the specifics of the application's URL handling, and the structure/architecture of the app. It seems reasonable to me to have a sensible default with the ability to override it if necessary though, particularly as `Path` is a configurable value, and these two go hand-in-hand.

Reply:
I think it's YAGNI but if you'd prefer to leave it in then fine by me 👍